pep517/build.py

```diff
@@ -31,7 +32,7 @@ def load_system(source_dir):
     Load the build system from a source dir (pyproject.toml).
     """
     pyproject = os.path.join(source_dir, 'pyproject.toml')
-    with open(pyproject) as f:
+    with io.open(pyproject, encoding="utf-8") as f:
```

Comment:
`io.open` is necessary for Python 2, where `builtins.open` lacks an "encoding" argument.